frontend/DataView/DataView.js

```diff
@@ -195,25 +195,31 @@ class DataItem extends React.Component<Props, State> {
   }
 
   inspect() {
+    if (this.props.inspect === null) {
+      return;
```

Comment:
Might be worth commenting why this is useful. Someone later might look at this code and think: "I'll just simplify this by passing an empty function!"